frontend/src/app/main/ui/shapes/shape.cljs

```diff
@@ -63,9 +63,12 @@
       [:defs
        [:& defs/svg-defs          {:shape shape :render-id render-id}]
        [:& filters/filters        {:shape shape :filter-id filter-id}]
-       [:& grad/gradient          {:shape shape :attr :fill-color-gradient}]
        [:& grad/gradient          {:shape shape :attr :stroke-color-gradient}]
-       [:& fim/fill-image-pattern {:shape shape :render-id render-id}]
+       (when (or (some? (:fill-image shape))
+               (= :image (:type shape))
+               (> (count (:fills shape)) 1)
+               (some #(some? (:fill-color-gradient %)) (:fills shape)))
```

Comment:
You can just use `(some :fill-color-gradient)`

Reply:
Do you refer just to the last condition so it looks like this?
```
(some :fill-color-gradient (:fills shape))
```

Reply:
Yes